Dockerfile

```diff
@@ -83,6 +83,7 @@ COPY patches ./patches
 FROM yarn-base as yarn-deps
 
 RUN --mount=type=cache,target=/usr/local/share/.cache \
+  --mount=type=cache,target=/root/.cache/Cypress \
```

Comment:
Cypress changes look like scope creep for the storybooks issue, this should probably have their own PR.

Reply:
Agreed! There are three separate changes in this PR.

* Removing storybook
* Removing wait-on (and a few extra packages)
* Cypress cache changes

I will create a PR for the Cypress changes and keep the package removal in this PR as separate commits.